packages/neuron-wallet/src/exceptions/common.ts

```diff
@@ -40,11 +40,18 @@ export class ShouldBeTypeOf extends Error {
   }
 }
 
+export class InvalidJSON extends Error {
+  constructor() {
+    super(i18n.t('messages.invalid-JSON'))
```

Comment:
We can use `small-case` key even if `JSON` is a proper noun.

Reply:
updated in https://github.com/nervosnetwork/neuron/pull/864/commits/227448f041be587a3df49849fd77e0833ee4b9c7